components/FlexibleHeader/examples/supplemental/FlexibleHeaderTopLayoutGuideSupplemental.m

```diff
@@ -49,7 +49,7 @@ - (void)setupScrollViewContent {
   UIColor *color =
       [UIColor colorWithRed:(97.0 / 255.0) green:(97.0 / 255.0) blue:(97.0 / 255.0) alpha:1.0];
   UIView *scrollViewContent =
-      [[UIView alloc] initWithFrame:CGRectMake(0, 0, self.scrollView.frame.size.width, 700)];
+      [[UIView alloc] initWithFrame:CGRectMake(0, 0, self.scrollView.frame.size.width, 1000)];
```

Comment:
Nit/suggestion: Does it make more sense to grab the main screen and use double the max of its height/width or something?  It seems like this is a fix due to iPhone X being taller than iPhone *Plus.  Does this still scroll on an iPad Pro (1366 points)?